src/test/java/ro/unibuc/hello/controller/GreetingsControllerIntegrationTest.java

```diff
@@ -36,10 +36,7 @@ public class GreetingsControllerIntegrationTest {
     @Container
     public static MongoDBContainer mongoDBContainer = new MongoDBContainer("mongo:6.0.20")
             .withExposedPorts(27017)
-            .withEnv("MONGO_INITDB_ROOT_USERNAME","root") // user
-            .withEnv("MONGO_INITDB_ROOT_PASSWORD", "example") // password
-            .withEnv("MONGO_INITDB_DATABASE", "testdb") // dbname
-            .withCommand("--auth");
+            .withSharding();
```

Comment:
Without this, Mongo was trying to use replica sets and I couldn't make that work. I think this creates a one-shard cluster, so basically a no-op.